Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml

```diff
@@ -585,3 +585,25 @@
       effects:
       - !type:SatiateThirst
         factor: 0.6
+
+- type: reagent
+  id: Chamomile
+  name: reagent-name-chamomile
+  parent: BaseDrink
+  desc: reagent-desc-chamomile
+  physicalDesc: reagent-physical-desc-aromatic
+  flavor: chamomile
+  color: "#27133b"
+  recognizable: true
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 2
+      - !type:GenericStatusEffect
+        key: Drowsiness
```

Comment:
I tested this in-game and it didn't have any drowsiness effect, likely because the status gets refreshed each time you take a sip. I'd recommend just removing the effect rather than making it work though, easy access to budget chloral hydrate doesn't seem like a good idea